manager-bundle/src/HttpKernel/ContaoKernel.php

```diff
@@ -394,11 +394,15 @@ private static function loadEnv(string $projectDir): void
             return;
         }
 
-        if (!file_exists($projectDir.'/.env')) {
-            return;
+        // Load cached env vars if the .env.local.php file exists
+        // See https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/4.2/config/bootstrap.php
+        if (\is_array($env = @include $projectDir.'/.env.local.php')) {
+            foreach ($env as $k => $v) {
+                $_ENV[$k] = $_ENV[$k] ?? (isset($_SERVER[$k]) && 0 !== strpos($k, 'HTTP_') ? $_SERVER[$k] : $v);
+            }
+        } elseif (file_exists($projectDir.'/.env')) {
```

Comment:
Please do not check for that file here, as the `DotEnv` component does the same (but also loads a `.env.dist` if available).

Reply:
I did not add the `file_exists()` check. It is already there in Contao 4.8:

https://github.com/contao/contao/blob/c3bff832feff0d4706dd5626dfca670dd595fa53/manager-bundle/src/HttpKernel/ContaoKernel.php#L400-L402

Reply:
Removed in 3092d0dea687b47e4e545fd70b6696c93b1f9bc1.

Reply:
It seems the check is needed: https://travis-ci.com/contao/contao/jobs/238118452

Reply:
Re-added in b6edf0f56db2221156cb390d6ef3ddbfc895805c.